include/zephyr/bluetooth/gap.h

```diff
@@ -21,369 +21,693 @@ extern "C" {
-#define BT_APPEARANCE_HEARING_AID_IN_EAR               0x0A41 /* In-ear hearing aid */
-#define BT_APPEARANCE_HEARING_AID_BEHIND_EAR           0x0A42 /* Behind-ear hearing aid */
-#define BT_APPEARANCE_HEARING_AID_COCHLEAR_IMPLANT     0x0A43 /* Cochlear Implant */
-#define BT_APPEARANCE_GENERIC_GAMING                   0x0A80 /* Generic Gaming */
-#define BT_APPEARANCE_HOME_VIDEO_GAME_CONSOLE          0x0A81 /* Home Video Game Console */
-#define BT_APPEARANCE_PORTABLE_HANDHELD_CONSOLE        0x0A82 /* Portable handheld console */
-#define BT_APPEARANCE_GENERIC_SIGNAGE                  0x0AC0 /* Generic Signage */
-#define BT_APPEARANCE_SIGNAGE_DIGITAL                  0x0AC1 /* Digital Signage */
-#define BT_APPEARANCE_SIGNAGE_ELECTRONIC_LABEL         0x0AC2 /* Electronic Label */
-#define BT_APPEARANCE_GENERIC_PULSE_OXIMETER           0x0C40 /* Generic Pulse Oximeter */
-#define BT_APPEARANCE_PULSE_OXIMETER_FINGERTIP         0x0C41 /* Fingertip Pulse Oximeter */
-#define BT_APPEARANCE_PULSE_OXIMETER_WRIST             0x0C42 /* Wrist Worn Pulse Oximeter */
-#define BT_APPEARANCE_GENERIC_WEIGHT_SCALE             0x0C80 /* Generic Weight Scale */
-#define BT_APPEARANCE_GENERIC_PERSONAL_MOBILITY_DEVICE 0x0CC0 /* Generic Personal Mobility Device */
-#define BT_APPEARANCE_MOBILITY_POWERED_WHEELCHAIR      0x0CC1 /* Powered Wheelchair */
-#define BT_APPEARANCE_MOBILITY_SCOOTER                 0x0CC2 /* Mobility Scooter */
-#define BT_APPEARANCE_CONTINUOUS_GLUCOSE_MONITOR       0x0D00 /* Continuous Glucose Monitor */
-#define BT_APPEARANCE_GENERIC_INSULIN_PUMP             0x0D40 /* Generic Insulin Pump */
-#define BT_APPEARANCE_INSULIN_PUMP_DURABLE             0x0D41 /* Insulin Pump, durable pump */
-#define BT_APPEARANCE_INSULIN_PUMP_PATCH               0x0D44 /* Insulin Pump, patch pump */
-#define BT_APPEARANCE_INSULIN_PEN                      0x0D48 /* Insulin Pen */
-#define BT_APPEARANCE_GENERIC_MEDICATION_DELIVERY      0x0D80 /* Generic Medication Delivery */
-#define BT_APPEARANCE_GENERIC_SPIROMETER               0x0DC0 /* Generic Spirometer */
-#define BT_APPEARANCE_SPIROMETER_HANDHELD              0x0DC1 /* Handheld Spirometer */
-#define BT_APPEARANCE_GENERIC_OUTDOOR_SPORTS           0x1440 /* Generic Outdoor Sports Activity */
-#define BT_APPEARANCE_OUTDOOR_SPORTS_LOCATION          0x1441 /* Location Display */
-#define BT_APPEARANCE_OUTDOOR_SPORTS_LOCATION_AND_NAV  0x1442 /* Location and Navigation Display */
-#define BT_APPEARANCE_OUTDOOR_SPORTS_LOCATION_POD      0x1443 /* Location Pod */
-#define BT_APPEARANCE_OUTDOOR_SPORTS_LOCATION_POD_AND_NAV 0x1444 /* Location and Navigation Pod */
-
-/* Defined GAP timers */
+/**
+ * @name Company Identifiers (see Bluetooth Assigned Numbers)
+ * @{
+ */
+#define BT_COMP_ID_LF           0x05f1 /**< The Linux Foundation */
+/**
+ * @}
+ */
+
+/**
+ * @name EIR/AD data type definitions
+ * @{
+ */
+#define BT_DATA_FLAGS                   0x01 /**< AD flags */
```

Comment:
Does the `<` indicate that this comment applies to the value coming before it?

Reply:
Yep! https://www.doxygen.nl/manual/docblocks.html#memberdoc